app/routes/public/sessions.js

```diff
@@ -66,6 +72,31 @@ export default class SessionsRoute extends Route {
       });
     }
 
+    if (params.track) {
+      filterOptions.push({
+        name : 'track',
+        op : 'has',
+        val : {
+          name: 'name',
+          op: 'eq',
+          val: params.track
+        }
+      })
+    }
+
+    if (params.room) {
+      filterOptions.push({
+        name : 'microlocation',
+        op : 'has',
+        val : {
+          name: 'name',
+          op: 'eq',
+          val: params.room
+        }
+      })
+    }
+  
```

Comment:
Codacy found an issue: [Trailing spaces not allowed.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6072219)